src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -139,15 +139,19 @@ public static final class DescriptorImpl extends ScanDescriptor {
 		
 		@Override
 		public String getDisplayName() {
-			return STATIC_ANALYZER;
+			return "Static Analysis (SAST)";
 		}
 
 		public FormValidation doCheckOpenSourceOnly(@QueryParameter Boolean openSourceOnly, @RelativePath("..") @QueryParameter String credentials, @AncestorInPath ItemGroup<?> context) {
             		JenkinsAuthenticationProvider checkAppScan360Connection = new JenkinsAuthenticationProvider(credentials,context);
-			if((openSourceOnly && checkAppScan360Connection.isAppScan360())) {
-                            return FormValidation.error(Messages.error_sca_ui());
-                	}
-                return FormValidation.ok();
+			if(openSourceOnly) {
```

Comment:
Can you correct the indentation here? It's confusing to read because the nested "else" case is at the same indentation as the outer "if" case.

Reply:
Okay.